app/lib/resources.server.ts

```diff
@@ -8,6 +8,8 @@ import type { Octokit } from "octokit";
 
 export type CacheContext = { octokit: Octokit };
 
+const GITHUB_URL = "https://github.com";
```

Comment:
I left out the "/", not sure what your preference is on this one @brookslybrand

Reply:
Doesn't super matter to me. Thanks for doing this!